common/src/WooCommerce.php

```diff
@@ -220,4 +246,130 @@ private function statusReached(string $new_status): bool {
         }
         return false;
     }
-}
+
+    private function get_product_data(array $order_arr) {
+        $pf = new WC_Product_Factory();
+        $products = [];
+        foreach ($order_arr['line_items'] as $line_item) {
+            $product = $pf->get_product($line_item['product_id']);
+            if (!$product) {
+                continue;
+            }
+            $gtin_handler = new GtinHandler($this->plugin);
+            $gtin_handler->setProduct($product);
+            $products[] = [
+                'id' => $product->get_id(),
+                'name' => $product->get_name(),
+                'url' => get_permalink($product->get_id()),
+                'image_url' => $this->getProductImage($product),
+                'sku' => $product->get_sku(),
+                'gtin' => $gtin_handler->getGtin(),
+                'reviews_allowed' => $product->get_reviews_allowed(),
+            ];
+        }
+        return $products;
+    }
+
+    private function getProductImage(WC_Product $product) {
+        foreach (get_attached_media('image', $product->get_id()) as $image) {
+            return wp_get_attachment_image_src($image->ID, 'full')[0] ?? null;
+        }
+    }
+
+    public function syncReviews(): void {
+        if (!get_option($this->plugin->getOptionName('product_reviews'))
+            || !$this->plugin->isWoocommerceActivated()
+        ) return;
```

Comment:
Please use braces `{`/`}` like everywhere else to make this more readable and consistent.